cypress/e2e/debug/filterTable.cy.ts

```diff
@@ -85,5 +85,14 @@ describe('Tests for table filter', () => {
     cy.get('[data-cy-debug="clear-filter-btn"]').click()
     cy.get('[data-cy-debug="tableFilter"]').eq(3).type('*Simple*');
     cy.assertDebugTableLength(2);
+  });
+
+  it('Should convert filter to new values when a different view is selected with different columns', () => {
+    cy.get('[data-cy-debug="filter"]').click();
+    cy.get('[data-cy-debug="tableFilter"]').eq(1).type('1{enter}');
```

Comment:
Value '1' is not a valid EndTime value for this filter right?

Reply:
It is. As long as it follows part of the pattern of endtime, it should not give an error. Validation for endtime is bit finicky as the use of wildcards was also required.

This means if the date is 13-12-2024 24:30:25 and I put 13-12 or just 13 as the filter it should recognise it as well as part of the date.